ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -81,10 +80,10 @@ const AdvancedGasFeeDefaults = () => {
     >
       <label className="advanced-gas-fee-defaults__label">
```

Comment:
We may be able to reduce some of the jsx here and use the `Checkbox` component API for the label https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-checkbox--docs#label